selectivecar.go

```diff
@@ -238,21 +241,22 @@ func (sct *selectiveCarTraverser) loader(lnk ipld.Link, ctx ipld.LinkContext) (i
 }
 
 func (sct *selectiveCarTraverser) traverseBlocks() error {
-
-	nsc := dagpb.AddDagPBSupportToChooser(func(ipld.Link, ipld.LinkContext) (ipld.NodePrototype, error) {
+	nsc := func(lnk ipld.Link, lctx ipld.LinkContext) (ipld.NodePrototype, error) {
+		// We can decode all nodes into basicnode's Any, except for
+		// dagpb nodes, which must explicitly use the PBNode prototype.
+		if lnk, ok := lnk.(cidlink.Link); ok && lnk.Cid.Prefix().Codec == 0x70 {
+			return dagpb.Type.PBNode, nil
+		}
 		return basicnode.Prototype.Any, nil
```

Comment:
As I read this, a raw node will get built with an Any node, rather than a Bytes node? Will that matter?

Reply:
As far as I could tell, your RawNode was the same as a basicnode.Bytes at the codegen level, so it seemed unnecessary to expose an identic type in go-codec-dagpb again.

I'm pretty sure that the use of Any shouldn't incur an inefficiency, as it barely does much more than the Bytes prototype: https://github.com/ipld/go-ipld-prime/blob/6de9f5cbb6bd0d00246dac8cae896e8cb175029d/node/basic/any.go#L135-L142

But if the measurements prove me wrong, it wouldn't be hard to re-add a case here to return basicnode.Prototype.Bytes :)